CRM/Member/Import/Parser/Membership.php

```diff
@@ -461,7 +461,7 @@ protected function setFieldMetadata(): void {
    * @return string[]
    */
   protected function getOddlyMappedMetadataFields(): array {
-    $uniqueNames = ['membership_id', 'membership_contact_id'];
+    $uniqueNames = ['membership_id', 'membership_contact_id', 'membership_start_date', 'membership_join_date', 'membership_end_date', 'membership_source', 'member_is_override', 'member_is_test', 'member_is_pay_later', 'member_campaign_id'];
```

Comment:
why member_is_test member_x not membership?

Reply:
I took it from the `Membership.xml`